WordPress/src/main/java/org/wordpress/android/models/Theme.java

```diff
@@ -32,7 +30,7 @@
     private String mBlogId;
     private boolean mIsCurrent;
 
-    public static Theme fromJSONV1_1(JSONObject object, SiteModel site) throws JSONException {
+    public static Theme fromJSONV1_1(JSONObject object, long siteId) throws JSONException {
```

Comment:
I personally like this change, but I think the decision is that we are passing `SiteModel` instead of the its id to avoid confusion between local and remote ids.